snapshot-labs/sx-monorepo
# feat: new affix component

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/191

### How to test

1. Go to a proposal page
2. The sidebar should scroll with the main page
3. Sidebar is always visible, and follow the same affix behavior than twitter

### TODO 

- [x] Fix issue where the main content (left of sidebar) width is not dynamic, making the page wider than 100%
- [x] Fix issue when scrolling votes table with infinite scroll
